tutorials/deploy-xenforo-to-compute-engine/index.md

```diff
@@ -0,0 +1,267 @@
+
+	```bash
+	sudo postmap sasl_password
+	sudo chmod 600 sasl_password.db
+	sudo rm sasl_password
+	```
+
+5. Lastly, restart the postfix service:
+
+	```bash
+	sudo service postfix restart
+	```
+
+	And now your XenForo website can use the third-party service you choose to send emails more securely and efficiently.
+
+## Update Your Domain (Optional)
+
+Anyone can now use the external IP of your Compute Engine instance to access your XenForo website. To assign a domain name to this IP address, follow the steps below:
+
+1. Assign a static IP to your Compute Engine instance:
+
+	Currently the external IP you see on the [VM Instances](https://console.cloud.google.com/compute/instances) page changes every time you restart the instance. You need to assign a static IP address to it to make sure that your website is always accessible even if the server restarts. Go to the [External IP addresses](https://console.cloud.google.com/networking/addresses/list) page, and click **Ephemeral** in the row of the Compute Engine instance hosting your XenForo website. In the drop-down menu, choose **Static**. Enter a new name for this static address and click **Reserve**.
+
+	If you have Cloud SDK installed in the system, you can also [use the gcloud command-line to reserve a static IP address](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-external-ip-address).
+
+2. If you haven’t done so, get a domain name from domain name registrars. Follow their instructions to add the IP address you just reserved to your domain. Your website should be accessible with the domain name in a short while.
+
+## Cleaning Up
+
+After you have finished this tutorial, you can clean up the resources you created on Google Cloud Platform so that you will not be billed for them in the future. To clean up, you can delete the whole project or stop the Compute Engine instance.
+
+### Deleting the Project
+
+Go to the [Manage resources](https://console.cloud.google.com/cloud-resource-manager) page. Select the project you used for this tutorial, and click **Delete**. Note that once the project is deleted, the project ID cannot be reused.
+
+If you have Cloud SDK installed in the system, you can also [use the gcloud command-line to delete a project](https://cloud.google.com/sdk/gcloud/reference/projects/delete).
+
+### Stopping the Compute Engine Instance
+
+Go the [VM Instances](https://console.cloud.google.com/compute/instances) page. Select the instance you used for this tutorial, and click **Stop**. If you have reserved a static IP address, it charges you at an hourly rate when the address is not used. To release the address, go to [External IP addresses](https://console.cloud.google.com/networking/addresses/list) page, select the address and click **Release Static Address**. You may also want to go to the Cloud Storage [browser](https://console.cloud.google.com/storage/browser) page and delete the bucket you created during the tutorial.
```

Comment:
"Visit the [VM Instances](https://console.cloud.google.com/compute/instances) menu. Select the instance you used for this tutorial and click **Stop**. If you have reserved a static IP address, you are charged at an hourly rate when the address is not used. To release the address, go to [External IP addresses](https://console.cloud.google.com/networking/addresses/list) menu, select the address, and click **Release Static Address**. You might also want to go to the Cloud Storage [browser](https://console.cloud.google.com/storage/browser) page and delete the bucket you created during the tutorial"